public/manifest.json

```diff
@@ -2,14 +2,9 @@
   "short_name": "Pipeline editor",
   "name": "Cloud Pipelines - Pipeline editor",
   "icons": [
-    {
-      "src": "logo48.png",
-      "sizes": "48x48",
-      "type": "image/png"
-    },
     {
       "src": "favicon.ico",
-      "sizes": "48x48",
+      "sizes": "64x64 32x32 24x24 16x16",
```

Comment:
We might need to check that this works well in practice on Windows/Linux/MacOS/iOS/Android.